SA/SA.cna

```diff
@@ -1145,6 +1168,29 @@ beacon_command_register(
 	"Synopsis: netsession <computer> "
 );
 
+alias netsession2 {
+	local('$args $hostname $method $dnsserver');
+
+	$hostname = iff(-istrue $2, $2, "");
+	$method = iff(-istrue $3, $3, 0);
+	$dnsserver = iff(-istrue $4, $4, "");
+
+	$args = bof_pack($1, "Zsz", $2, $3, $4);
+	beacon_inline_execute($1, readbof($1, "get-netsession2", $null, "T1049"), "go", $args);     #T1049 is a loose match
+}
+
+beacon_command_register(
+	"netsession2",
+	"list sessions on server. Output is compatible with bofhound",
+	"Usage: netsession2 <opt: computer> <opt: Resolution Method> <opt: DNS Server>
+	Resolution methods: 
+		1 = DNS (Default)
```

Comment:
This says 1 is the default, but 0 is the default used when this argument isn't provided and 0 is the value checked for in the bof

Reply:
Great catch, somehow I messed this up in a way that sort of worked. Fixed the CNA to default `1` and the BOF to check for `1`